pegasus/sites.v3/code.org/public/teacher-dashboard/section_assessments.haml

```diff
@@ -11,7 +11,7 @@ content-type: text/ng-template
 
 %div{'ng-show' => 'assessmentsLoaded'}
   = I18n.t('dashboard_assessments_view')
-  %select{"ng-model" => "script_id", "ng-options"=>"script.id as script.name group by script.category for script in script_list | orderBy:['category_priority', 'category', 'position', 'name']", 'ng-change' => 'changeScript(script_id)'}
+  %select#course-dropdown{"ng-model" => "script_id", "ng-options"=>"script.id as script.name group by script.category for script in script_list | orderBy:['category_priority', 'category', 'position', 'name']", 'ng-change' => 'changeScript(script_id)'}
```

Comment:
My approach for ids that are used only in tests has been to give them a name of `uitest-foo`.

My reasoning for doing this is when I see this code as it stands right now, I need to worry about whether `course-dropdown` is used to (a) apply stylings through CSS and/or (b) we have jquery using this id to do DOM manipulations somewhere. If I see `uitest-course-dropdown` I feel safer in assuming we dont have these external usages, and that this is just needed for UI tests.

Thoughts?